Lib/test/test_ftplib.py

```diff
@@ -24,9 +24,9 @@
 TIMEOUT = support.LOOPBACK_TIMEOUT
 # the dummy data returned by server over the data channel when
 # RETR, LIST, NLST, MLSD commands are issued
-RETR_DATA = 'abcde12345\r\n' * 1000
-LIST_DATA = 'foo\r\nbar\r\n'
-NLST_DATA = 'foo\r\nbar\r\n'
+RETR_DATA = 'abcde12345\r\n' * 1000 + 'special char ®\r\n'
```

Comment:
Your change is correct. But usually I prefer to avoid non-ASCII characters in the Python source code to prevent developers mojibake issue if your operating system or editor don't support the character. I prefer use \xHH or \uHHHH to keep the source code written in pure ASCII. I suggest:

```suggestion
RETR_DATA = 'abcde12345\r\n' * 1000 + 'non-ascii char \xAE\r\n'
```

(and do a similar change for other constants and for expected output.)